### PR TITLE
CPA-97 Drag-n-drop image

### PR DESCRIPTION
**Pull request by KenAdams777**
Implemented grad&drop uploader for an image.
Feel free to leave your comments and suggestions.

### CODE REVIEW

**Review by MuscularSloth — COMMENTED** [review]
As I see we have here Delete Comment logic too. Do we need to merge your first PR before review this one?

**Comment by KenAdams777** [comment]
> As I see we have here Delete Comment logic too. Do we need to merge your first PR before review this one?

I noticed it as well.
I guess we do.

**Review by MuscularSloth — APPROVED** [review]

File: src/components/Comment/Comment.js [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+	const [currentCommentId, setCurrentCommentId] = useState(null);
+
 	const date = useDate(comment.created);
 
+	useEffect(() => {
+		(isDeletedSuccessfully || deleteCommentError) && setCurrentCommentId(null);
+	}, [isDeletedSuccessfully, deleteCommentError]);
+
+	const handleDelete = () => {
+		if (window.confirm("Are you sure?")) {
```
Comment (bejalane) [inline src/components/Comment/Comment.js:25]: it should be simple MUI modal and not browser modal

**Review by bejalane — APPROVED** [review]
Great job!

File: src/components/Comment/Comment.js (hunk 1 above)
Comment (KenAdams777) [reply src/components/Comment/Comment.js]: > it should be simple MUI modal and not browser modal

Fixed.

**Pull request merged**